arviz/plots/ppcplot.py

```diff
@@ -124,7 +124,7 @@ def plot_ppc(
 
         >>> import arviz as az
         >>> data = az.load_arviz_data('radon')
-        >>> az.plot_ppc(data)
+        >>> az.plot_ppc(data,data_pairs={"obs":"obs"})
```

Comment:
This looks good. Maybe we could add an example from `pystan` case too, e.g. comment it out
```
>>> # az.plot_ppc(data, data_pairs={"obs" : "obs_hat"})
```
cc @OriolAbril

Reply:
Also, maybe we could add 4 spaces to indent the lines 65, 66 and make key and value lowercase.

```
Dictionary structure:
    key = data var_name
    value = posterior predictive var_name
```

Reply:
I have updated the file with the required lines and changes. However, I am a little unfamiliar with the parameters `"obs"` and `"obs_hat"`. Could you please link me to documentation that may help me learn more about these?